### PR TITLE
Hotfix/scoring

### PR DESCRIPTION
**Pull request by DavidWilliamK**

### CODE REVIEW

**Comment by karnandohuang** [comment]
unit tests

File: src/views/Quiz/QuizDetail.vue [hunk 1]
```diff
@@ -41,7 +41,8 @@
               </div>
               <div class="tile is-parent">
                 <article class="tile is-child box">
-                  <p>Time Limit</p>
+                  <span>Time Limit</span>
+                  <span style="font-size: 0.6rem">({{isMinutes ? 'Minutes' : 'Hours'}})</span>
```
Comment (oliverrsebastian) [inline src/views/Quiz/QuizDetail.vue]: udh dicoba di mobile kah?

**Comment by oliverrsebastian** [comment]
Please check the style for all display type @DavidWilliamK

File: src/views/Quiz/QuizDetail.vue (hunk 1 above)
Comment (DavidWilliamK) [reply src/views/Quiz/QuizDetail.vue]: Sudahh
image

Ada issue kah? @oliverrsebastian

Comment (oliverrsebastian) [reply src/views/Quiz/QuizDetail.vue]: itu antara 2 span bisa dikasih margin ga vid biar ga nempel gitu wkwkwk

File: src/components/modals/ModalSelectQuestionBank.vue [hunk 2]
```diff
@@ -8,12 +8,12 @@
             <span class="modal__close" @click="close"><b-icon icon="times" size="lg"></b-icon></span>
           </div>
           <div class="modal__body">
-            <div class="columns is-multiline" v-if="isLoading">
-              <div class="column is-12" v-for="n in 3" :key="n">
-                <ListItem :minHeight="'50px'" :simple="true" :loading="isLoading"></ListItem>
+            <div v-if="!isLoading" class="columns is-mobile">
+              <div class="column is-flex">
+                <b-checkbox v-model="allSelected" :indeterminate="partialSelected">Select all question bank</b-checkbox>
```
Comment (oliverrsebastian) [inline src/components/modals/ModalSelectQuestionBank.vue]: `indeterminate` means if `partialSelected` is true, then the v-model will follow `partialSelected` right?

File: src/components/modals/js/modal-select-question-bank.js [hunk 3]
```diff
@@ -30,7 +26,21 @@ export default {
   computed: {
     ...mapGetters([
       'questionBanks'
-    ])
+    ]),
+    partialSelected () {
+      return (this.selectedId.length !== this.questionBankList.length) && this.selectedId.length > 0
```
Comment (oliverrsebastian) [inline src/components/modals/js/modal-select-question-bank.js]: if don't mind, please refactor this name to `selectedIdList`

**Pull request merged**